src/ACEtk/details/verify/src/size.hpp

```diff
@@ -0,0 +1,10 @@
+template< typename R1, typename R2 >
+inline void equalSize( R1&& a1, R2&& a2 ){
+  if( ranges::distance( a1 ) != ranges::distance( a2 ) ){
+    njoy::Log::error( "array sizes are not the same" );
```

Comment:
This error message is vague. Let's take two `const char*` arguments in the function to specifying names to provide a better error message.

Reply:
In order to keep this method simpler, I would rather defer more descriptive messages later down the line.

Reply:
The current way is more consistent with the other verifications in ACEtk. 

I did change the error message a bit.

Reply:
You'd prefer that, at each call site, you add logic to specifically catch an range-error exception to specify the names of the mismatched ranges? That's an API that's easy to use poorly.